R/read_csv.R

```diff
@@ -370,9 +373,112 @@ read_sample_csv <- function(files,
   read_cmdstan_csv(files, variables, sampler_diagnostics)
 }
 
+#' @rdname read_cmdstan_csv
+#' @export
+#' @param csv_contents The list returned by `read_cmdstan_csv()`.
+#' @param check_diagnostics Should diagnostic checks be performed? The default
+#'   is `TRUE` but set to `FALSE` to avoid checking diagnostics like
+#'   divergences.
+#' @examples
+#' \dontrun{
+#' # create a CmdStanMCMC object
+#' x <- read_cmdstan_csv(fit$output_files())
+#' fit <- as_cmdstan_mcmc(x)
+#' }
+as_cmdstan_mcmc <- function(csv_contents, check_diagnostics = TRUE) {
```

Comment:
What do you think of the names `as_cmdstan_mcmc()`, `as_cmdstan_mle()`, `as_cmdstan_vb()`?

Reply:
I like them, but have another idea. What about `as_cmdstan_fit()` that does all three?

Reply:
Good idea. I just changed it.